test/util/race.go

```diff
@@ -7,6 +7,8 @@ import (
 	"testing"
 )
 
+const _flakyTestEnvKey = "SKIP_FLAKY_TESTS"
```

Comment:
stylenit:
```suggestion
const flakyTestEnvKey = "SKIP_FLAKY_TESTS"
```

Reply:
I just noticed the commit suggestion dropdown. I already fixed it locally.